pkg/websocket/server.go

```diff
@@ -17,6 +17,7 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+// wsDialer is the default dialer but with a shorter timeout.
```

Comment:
i removed this comment in this pr because it's not accurate :). as this dialer doesn't use a proxy either.